docs/development/install.rst

```diff
@@ -90,37 +184,51 @@ you need these libraries.
 Get and run Read the Docs
 -------------------------
 
-Clone the repository somewhere on your disk and enter to the repository::
+Clone the repository somewhere on your disk and enter to the repository:
+
+.. code:: console
```

Comment:
Using ``.. prompt:: bash`` avoids adding `$` to the lines (adds it automatically) and make them un-selectable.

Reply:
The latest version of the theme should do this. `code:: prompt` did not give the same results when I tried.